### PR TITLE
Add io.neovim.nvim

### PR DESCRIPTION
**Pull request by jurf**
- [x] AppData
 - [x] Testing
 - [x] Proper icons

etc.

But so far it works.

### CODE REVIEW

**Comment by jurf** [comment]
bot, build io.neovim.neovim

**Comment by barthalion** [comment]
- [ ] upstream involvement

**Comment by jurf** [comment]
@justinmk do you want write access to the Neovim Flathub repo, once it’s created? Or do you know any of the other devs who would if not?

**Comment by justinmk** [comment]
Thanks, I don't need write access but feel free to ping me as needed.

**Comment by nedrichards** [comment]
bot, build io.neovim.neovim

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
Oh come on! Why won’t it frickin’ [build](https://flathub.org/builds/#/builders/42/builds/415)? I’m at loss completely, and also tempted to just disable ARM 64…

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
@Erick555 yeah, suspected that, but unfortunately the lecture ended and I had to do some actual studying. :wink:

I noticed that issue but it kinda ended in a limbo, I’m glad it got somewhere now. I guess disabling ARM64 is probably the best solution for now?

Also, yeah, I checked the docs, but somehow completely missed it’s a boolean. >.< I thought I checked that specifically!

**Comment by Erick555** [comment]
> I noticed that issue but it kinda ended in a limbo, I’m glad it got somewhere now. I guess disabling ARM64 is probably the best solution for now?

Yeah, that's reasonable for now.

**Comment by jurf** [comment]
@justinmk, do you have any description for Neovim (suitable for [AppStream](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description)) at hand? Something like [this](https://flathub.org/apps/details/org.gnome.Recipes). I’ve looked through what is already available, but haven’t found anything which would fit the format.

**Comment by justinmk** [comment]
@jurf see the description at https://summerofcode.withgoogle.com/archive/2018/organizations/5474817897660416/

> Vim-fork focused on extensibility and usability
>
> Neovim is a refactor of the Vim text editor engineered to encourage new features, advanced UIs, and powerful extensions.

**Comment by justinmk** [comment]
when this is ready could you add a note to https://github.com/neovim/neovim/wiki/Installing-Neovim#linux

**Comment by jurf** [comment]
@justinmk hm, thanks, though it’s a tad short. If you haven’t anything longer I’ll try writing something up or leaving it be, it should ideally be 1–3 paragraphs, maybe a list of features, etc.

Sure, I’ll add the note. Sorry for the late reply, I’ve spent the last week full-time (literally) studying maths.

**Comment by jurf** [comment]
bot, build io.neovim.nvim

**Comment by jurf** [comment]
Added a basic AppData. Maybe a bit too basic. We could use some screenshots at least…

**Comment by jurf** [comment]
bot, build io.neovim.nvim

File: io.neovim.nvim.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <url type="faq">https://github.com/neovim/neovim/wiki/FAQ</url>
+  <url type="help">https://neovim.io/doc/</url>
+  <url type="donation">https://neovim.io/sponsors/</url>
+  <url type="translate">https://github.com/neovim/neovim/tree/master/src/nvim/po</url>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release date="2018-07-19" version="0.3.1"/>
+    <release date="2018-07-11" version="0.3.0"/>
+  </releases>
+  <launchable type="desktop-id">nvim.desktop</launchable>
```
Comment (x80486) [inline io.neovim.nvim.appdata.xml]: Does this works in the same way `io.neovim.nvim.desktop` would be?

Comment (jurf) [reply io.neovim.nvim.appdata.xml]: It gets renamed during along with the desktop file.

File: io.neovim.nvim.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline io.neovim.nvim.appdata.xml]: Has it been submitted upstream?

Comment (jurf) [reply io.neovim.nvim.appdata.xml]: Nope, forgot about that, will do.

File: io.neovim.nvim.appdata.xml (hunk 1 above)
Comment (barthalion) [reply io.neovim.nvim.appdata.xml]: Of course it should be renamed upstream too.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/io.neovim.nvim

**Comment by jurf** [comment]
Cheers.

**Pull request closed without merging**